spec/ddtrace/context_spec.rb

```diff
@@ -343,6 +343,27 @@ def new_unfinished_span(name = nil)
     end
   end
 
+  describe '#current_span_and_root_span' do
+    subject(:current_span_and_root_span) { context.current_span_and_root_span }
+
+    let(:span) { Datadog::Span.new(tracer, 'span', context: context) }
+    let(:root_span) { Datadog::Span.new(tracer, 'root span', context: context) }
+
+    it 'returns the current span as well as the current root span' do
+      context.add_span(root_span)
+      context.add_span(span)
+
+      current_span, current_root_span = current_span_and_root_span
+
+      expect(current_span).to be span
+      expect(current_span).to be context.current_span
+      expect(current_root_span).to be root_span
+      expect(current_root_span).to be context.current_root_span
+    end
+
+    # NOTE: The rest of the behavior tests for setting the root span is in #current_root_span
```

Comment:
I'm not sure what this note means: is it a reference to future work? Is it referring to the block above?

Reply:
I mean that this spec is a bit bare on what it means to be a root span and when does it change; but that the tests for this behavior are in `#current_root_span`. I guess if the note is confusing, maybe I'll just remove it...?

Thoughts?

Reply:
I think you can remove it. If it's not being tested here, I'd assume it's covered elsewhere.

The comment lets us know that we didn't forget to test, but I'm not sure if it's strictly needed here.

Reply:
👍  I'll remove this before merging